person.rb

```diff
@@ -1,8 +1,10 @@
+require_relative 'corrector'
+
 class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(age:, name: 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
The corrector class has not been instantiated for Person, see [reference](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes)

- [ ] Kindly create an instance of Corrector set in the Person class on initialization